src/server/rooms/SkyOffice.ts

```diff
@@ -0,0 +1,45 @@
+import { Room, Client } from 'colyseus'
```

Comment:
FYI: Again this file is really a "rename" of MyRoom.ts, but it's shown as delete+newfile, use rename next time

Reply:
ok